openlibrary/plugins/openlibrary/js/jquery-ui/lazy.js

```diff
@@ -13,17 +13,7 @@ function init() {
 function placeholder(fnName) {
     return function () {
         const $this = $(this);
-        // only if the selector matches load the additional code and wire it up.
-        if ($this.length) {
-            init().then(() => {
-                if (fnName === 'colorbox') {
-                    // set option to open immediately since loading was delayed.
-                    arguments[0].open = true;
-                }
-                // apply it for real now this function has been replaced
-                $this[fnName].apply(this, arguments);
-            });
-        }
+        init().then(() => $.fn[fnName].apply($this || this, arguments));
```

Comment:
The length check was important as it makes sure this doesn't run on elements that do not exist in the DOM.

One goal of this is to drop jquery-ui from loading on the home page where it's not needed yet with the latest patch I'm seeing it loaded in full.

In the current code there are unguarded calls to enhance all the following elements:
#noMaster, #confirmMerge, #leave-waitinglist-dialog and #tabsAddbook,#tabsAddauthor,.tabs:not(.ui-tabs), .cta-btn--preview

yet none of them are on a book page for example.